src/main.py

```diff
@@ -26,10 +26,11 @@
 
 # Initialize helper classes
 dbg = Debug(config.OWNER, config.CMD_PREFIX, config.DEBUG_BOT)
-spam = Spammers()
-tk = Timekeeper()
-watch = Watcher()
 frwrdr = MessageForwarder(config.MAILBOX, config.HOME_SERVER, config.THREAD_ROLES)
+spam = Spammers(frwrdr)
+tk = Timekeeper()
+watch = Watcher(frwrdr)
+commands.frwrdr = frwrdr
```

Comment:
Would it be possible/better to make the MessageForwarder instance part of the `client`? Then we only have a single instance and don't need to pass into every constructor. This "top down" approach seems like the opposite way to do it, but I'm not sure how feasible that is.

Reply:
Fair, yeah I didn't love it either.

I like having it in the client, then it's usable everywhere. The problem is it creates another dependency cycle: client imports forwarder, forwarder imports client. Also, if `discord.py` every introduces something of the same name we won't have access.

This gave me an idea: create a global instance in `forwarder.py`, and then import that everywhere as needed.

I think that works well: it's available everywhere (e.g. just like `client`), and there are no dependency cycles.

LMK what you think!